README.md

```diff
@@ -14,31 +14,26 @@
 
 ---
 
-## What is the Oisy Wallet
+## What is the Oisy wallet
```

Comment:
No more "Wallet" with a capital letter?
Should we update the code base to reflect this change?

Reply:
Let's do it gradually, but yes, I personally have a slight preference for `Oisy wallet` over `Oisy Wallet`